app-modules/audit/src/Providers/AuditServiceProvider.php

```diff
@@ -11,12 +14,19 @@ class AuditServiceProvider extends ServiceProvider
 {
     public function register()
     {
+        Panel::configureUsing(fn (Panel $panel) => $panel->plugin(new AuditPlugin()));
+
         app('config')->set('audit', require base_path('app-modules/audit/config/audit.php'));
+        app('config')->set('settings', require base_path('app-modules/audit/config/settings.php'));
```

Comment:
We might eventually run into some complications here, as I would think some other modules will potentially need to use Settings as well.

Reply:
Yeah, my plan was to possibly move this to a general Module at some point soon. Do you think anything would prevent us from moving it later?

Reply:
I don't think so - just wanted to keep it on our radar as it may benefit us to pivot to the other handling we looked into (hierarchical merging) for this case.